snippets/react-snippets.code-snippets

```diff
@@ -243,6 +243,15 @@
         ],
         "description": "Use `role='switch'` on checkboxes that represent an 'on' or 'off' state. Add `aria-checked` to indicate whether component is on or off. Add `aria-required` if the field is required."
     },
+    "CommentAria":{
+        "prefix": [
+            "CommentAria",
+        ],
+        "body": [
+            "role='comment'",
+            "aria-details={$1}"
```

Comment:
this parts tough because the `aria-details` actually should be located on role="mark"

```
<span role="mark" aria-details="thread-1">end of the guitar solo</span>, before fading away sharply.</p>

<div role="comment" id="thread-1" data-author="chris">
  <h3>Chris said</h3>
  <p class="comment-text">I really think this moment could use more cowbell.</p>
  <p><time datetime="2019-03-30T19:29">March 30 2019, 19:29</time></p>
</div>

```

Reply:
what we end up wanting is an id={} paired with role comment 
and a description explaining that when you use role=comment you also should use role=mark

Reply:
Updated the changes! Let me know what you think!